convert2rhel/utils/__init__.py

```diff
@@ -1115,3 +1116,20 @@ def write_json_object_to_file(path, data, mode=0o600):
     with open(path, mode="w") as handler:
         os.chmod(path, mode)
         json.dump(data, handler, indent=4)
+
+
+def warn_deprecated_env(env_name):
+    """Warn the user that the environment variable is deprecated.
+
+    :param env_name: The name of the environment variable that is deprecated.
+    :type env_name: str
+    """
+    if env_name not in os.environ:
+        # Nothing to do here.
+        return
+
+    root_logger.warning(
+        "The environment variable {} is deprecated and is set to be removed on Convert2RHEL 2.4.0.\n"
+        "Please, use the configuration file instead."
+    )
+    return True
```

Comment:
IMO create a new utils file and use that when introducing new code so we can migrate away from everything in `__init__.py` faster

Reply:
I didn't created a new one because this is about to be deprecated in a future release. We won't stick with this function very long, so I thought that creating a new file and etc would be just another step to remove it from the codebase later.

Reply:
Welll, either way, it would work as having the function here or new file. It will get removed in the future anyway.